onadata/libs/utils/dict_tools.py

```diff
@@ -63,7 +63,19 @@ def merge_list_of_dicts(list_of_dicts):
             else:
                 if k in result:
                     if isinstance(v, dict):
-                        result[k] = merge_list_of_dicts([result[k], v])
+                        try:
+                            result[k] = merge_list_of_dicts([result[k], v])
+                        except AttributeError:
+                            # If the key is within the override_keys
+                            # (Is a select_multiple question) We make
+                            # the assumption that the dict values are
+                            # more accurate as they usually mean that
+                            # the select_multiple has been split into
+                            # separate columns for each choice
+                            if isinstance(result[k], str) and \
+                                 k in override_keys:
+                                result[k] = {}
+                                result[k] = merge_list_of_dicts([result[k], v])
```

Comment:
What should happen in the else part of this check? Should we re-raise the exception or silently ignore as this currently does? Does it mean that an empty submission is made or the row is skipped?

Reply:
This would mean the row would be skipped & the previous value would be used on the field. I think we should re-raise the error and stop the CSV Import process here